api/pkg/apis/v1alpha1/providers/target/script/script_test.go

```diff
@@ -52,6 +52,7 @@ func TestInitMissingRemove(t *testing.T) {
 		"applyScript":   "b",
 	})
 	require.NotNil(t, err)
+	assert.Equal(t, err.Error(), "update error message")
```

Comment:
This is the wrong assert, here is the init error, not update error message error.